src/services/giphy.service.js

```diff
@@ -9,7 +9,8 @@
   /* @ngInject */
   function giphyService($http, giphyConfig) {
 
-    var baseUrl = 'http://api.giphy.com/v1/gifs';
+    var baseUrl = window.location.protocol === 'https' ? 'https://api.giphy.com/v1/gifs'
```

Comment:
What about this: 

```js
var baseUrl = window.location.protocol + '://api.giphy.com/v1/gifs'
```

Reply:
That was deliberate to account for the URLs that start with `file://`, `ftp://`, but anyway, I made a commit.

Reply:
Yeah I think it might be better to always use `https`